tests/test_code_echo.sh

```diff
@@ -293,3 +293,16 @@ echo
 echo -n "Testing C (Wrong Formatting)... "
 curl -F source=@$SOURCE localhost:9206/code/test-echo
 rm -f $SOURCE
+
+# PHP 7.2
+SOURCE=$(mktmp -t dredd_XXXXXXX.php)
```

Comment:
Should be mktemp

Move test to after Rust.

Reply:
Fixed